sriov/tests/conftest.py

```diff
@@ -44,8 +48,18 @@ def get_testdata_obj(settings: Config) -> ConfigTestData:
 
 
 @pytest.fixture
-def settings() -> Config:
-    return get_settings_obj()
+def settings(dut, trafficgen) -> Config:
```

Comment:
@dkosteck This is the key change for this PR, basically the we add the required pci info here, so that we can keep the configtestdata unchanged

Reply:
@dkosteck and because the configtestdata didn't change, so the test case didn't change